### PR TITLE
adding section on receiving data from Stripe

### PR DESCRIPTION
**Pull request by kuboschek**

### CODE REVIEW

File: pages/documents/privacy/2021.tsx [hunk 1]
```diff
@@ -0,0 +1,405 @@
...
+                <li>the amount paid, so we know what amount to print on the receipt;</li>
+                <li>the status of the payment, so we can determine if a payment was successful;</li>
+                <li>the currency that the payment was made in, so we can print the correct currency on the receipt;</li>
+            </ul>
+
+            <p>
+                Please also see the Privacy Policy of Stripe Inc. (
+                    <a href="https://stripe.com/de/privacy">https://stripe.com/de/privacy</a>
+                ) to read more details on how they ensure that your personal information is treated confidentially.
+            </p>
```
Comment (kuboschek) [inline pages/documents/privacy/2021.tsx]: This is the section that's being added.

File: pages/documents/privacy/2021.tsx [hunk 2]
```diff
@@ -0,0 +1,412 @@
...
+                <li>
+                    <ul>
+                        <li>your credit card details, i.e. your credit card number, expiration date and optionally your CVV (depending on the card type); or</li>
+                        <li>your SEPA bank account details, i.e. your bank account number (IBAN) and the owner of your Bank account.</li>
+                    </ul>
+                </li>
+            </ul>
+
+            <p>
+                The payment details are submitted directly from your browser to Stripe Inc. and the payment details are never stored by Jacobs University Alumni.
```
Comment (tkw1536) [inline pages/documents/privacy/2021.tsx]: "payment details are never stored by us".

File: pages/documents/privacy/2021.tsx [hunk 3]
```diff
@@ -0,0 +1,412 @@
...
+
+            <p>
+                The following received payment details are stored by Jacobs University Alumni:
+            </p>
+
+            <ul>
+                <li>a unique customer number, so that we know who made a payment;</li>
+                <li>the time the payment was made, so we know what date to print on the receipt;</li>
+                <li>the amount paid, so we know what amount to print on the receipt;</li>
+                <li>the status of the payment, so we can determine if a payment was successful;</li>
```
Comment (tkw1536) [inline pages/documents/privacy/2021.tsx]: instead of ";" write "; and" and end the last bullet with "."

**Review by tkw1536 — CHANGES_REQUESTED** [review]
In principle ok, minor changes needed.

File: pages/documents/privacy/2021.tsx (hunk 2 above)
Comment (kuboschek) [reply pages/documents/privacy/2021.tsx]: That is still true. We only store the payment history, not the payment details.

Comment (tkw1536) [reply pages/documents/privacy/2021.tsx]: Yes, I meant "Jacobs University Alumni" -> "us".

**Review by tkw1536 — APPROVED** [review]

**Pull request merged**